Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/Utils.java

```diff
@@ -369,6 +368,9 @@ public static Vector<ServerEntry> getXmlServerEntries(String xml) {
                     Node joinnode = joinlist.item(0);
                     if (joinnode.getNodeType() == Node.ELEMENT_NODE) {
                         Element joinelement = (Element) joinnode;
+                        NodeList rememberLastChannelnode = hostelement.getElementsByTagName("rememberLastChannel");
```

Comment:
can you rename it to "join-last-channel"? Then it matches the name from desktop client. Also other xml-tags are lower-case with dash.